raiden/tests/scenarios/ms1_simple_monitoring.yaml

```diff
@@ -34,7 +34,7 @@ nodes:
     proportional-imbalance-fee:
       - "0x59105441977ecD9d805A4f5b060E34676F50F806"
       - 0
-    default-settle-timeout: 40
+    default-settle-timeout: 100
```

Comment:
The `- wait_blocks: 40` below needs to be adjusted as well, the other scenarios as well.

Reply:
ah yes. good point.